### PR TITLE
Invalidate all parent skus if child sku is updated.

### PR DESCRIPTION
**Pull request by joshirohit100**

### CODE REVIEW

File: modules/acm_sku/src/Entity/SKU.php [hunk 1]
```diff
@@ -757,4 +757,20 @@ public function clearStockCache() {
...
+    $cache_tags = parent::getCacheTags();
+    /** @var \Drupal\acm_sku\AcquiaCommerce\SKUPluginBase $plugin */
+    $plugin = $this->getPluginInstance();
+    // Get parent skus(if any) for the sku.
+    $parent_skus = array_values($plugin->getAllParentSkus($this->getSku()));
+    // Prepare cache tags of parent sku.
+    $parent_skus = array_map(function($parent_sku) { return 'acm_sku:'. $parent_sku; }, $parent_skus);
+
+    // @Todo: Add the tags of the display node as well.
+    return Cache::mergeTags($cache_tags, $parent_skus);
```
Comment (nikunjkotecha) [inline modules/acm_sku/src/Entity/SKU.php:775]: this should be in getcachetagstoinvalidate, we don't want these cache tags all the time

Comment (joshirohit100) [reply modules/acm_sku/src/Entity/SKU.php:775]: getCacheTags() internally calls getcachetagstoinvalidate(). So its just semantics.

Comment (nikunjkotecha) [reply modules/acm_sku/src/Entity/SKU.php:775]: @joshirohit100 will Drupal use getCacheTags to invalidate when entity is saved?

Comment (nikunjkotecha) [reply modules/acm_sku/src/Entity/SKU.php:775]: check invalidateTagsOnSave in docroot/core/lib/Drupal/Core/Entity/Entity.php

Comment (nikunjkotecha) [reply modules/acm_sku/src/Entity/SKU.php:775]: adding missing use statement in https://github.com/acquia/commerce-manager/pull/126

**Pull request merged**